footprint: [[0.20, 0.20], [0.20, -0.20], [-0.20, -0.20], [-0.20, 0.20]]
footprint_padding: 0.01
global_frame: /map
robot_base_frame: base_link

static_map: false
resolution: 0.05
origin_x: -100.0
origin_y: -100.0

transform_tolerance: 0.5
publish_frequency: 1.0
update_frequency: 5.0

plugins:
- {name: static_layer, type: 'costmap_2d::StaticLayer'}
- {name: obstacle_layer, type: 'costmap_2d::ObstacleLayer'}
- {name: inflation_layer, type: 'costmap_2d::InflationLayer'}

inflation_layer: {cost_scaling_factor: 5.0, enabled: true, inflation_radius: 1.5}
obstacle_layer:
  combination_method: 1
  enabled: true
  footprint_clearing_enabled: true
  laser_scan_sensor: {clearing: true, data_type: LaserScan, expected_update_rate: 0.4,
    marking: true, sensor_frame: base_laser_link, topic: base_scan}
  max_obstacle_height: 2.0
  observation_sources: laser_scan_sensor
  obstacle_range: 1.5
  raytrace_range: 3.0
